example/cassandra/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express')
+const http = require('http')
+const cassandra = require('cassandra-driver')
+const terminus = require('../../lib/terminus')
+const app = express()
+const client = new cassandra.Client({contactPoints: ['127.0.0.1']})
+
+function onHealthCheck () {
+  return client.execute('SELECT now() FROM system.local')
+}
+
+async function onSignal () {
+  console.log('server is starting cleanup')
+  try {
+    await client.shutdown()
+    console.log('db has disconnected')
+  } catch (err) {
+    console.error('error during disconnection', err.stack)
+    throw err
+  }
+}
+
+client.connect()
+  .then(() => {
+    console.log('db connected')
+
+    terminus(http.createServer(app), {
+      logger: console.log,
+      signal: 'SIGINT',
```

Comment:
is this on-purpose? `SIGTERM` should be used for production environments, shouldn't it?

Reply:
I'm on Windows where `SIGTERM` is not supported, the same situation is in all examples.

Reply:
Oh, okay, makes sense